mne/source_estimate.py

```diff
@@ -447,13 +447,21 @@ class _BaseSourceEstimate(ToDataFrameMixin, TimeMixin):
 
     Parameters
     ----------
-    data : array, shape (n_dipoles, n_times) | tuple, shape (2,)
-        The data in source space. The data can either be a single array or
-        a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
-        "sens_data" shape (n_sensors, n_times). In this case, the source
-        space data corresponds to "numpy.dot(kernel, sens_data)".
-    vertices : array | list of array
-        Vertex numbers corresponding to the data.
+    data : array of shape (n_dipoles, n_times) | tuple, shape (2,)
+        The data in source space. When it is a single array, the
+        left hemisphere is stored in data[:vertices[0]] and the right
+        hemisphere is stored in data[:-vertices[1]].
+        When data is a tuple, it contains two arrays:
+
+        - "kernel" shape (n_vertices, n_sensors) and
+        - "sens_data" shape (n_sensors, n_times).
+
+        In this case, the source space data corresponds to
+        ``np.dot(kernel, sens_data)``.
+    vertices : array | list of shape (2,)
+        Vertex numbers corresponding to the data. The first element of the list
+        contains vertices of left hemisphere and the second element contains
+        vertices of right hemisphere.
```

Comment:
this is only true with you have a surface source space. If you have a volume source space or a mixed or discrete source space this is not correct.